README.md

```diff
@@ -79,9 +79,17 @@ end
 Resume iterating using the previously saved cursor.
 
 ```ruby
-Feed::Item.desc(:position).limit(5).scroll(saved_cursor) do |record, next_cursor|
+Feed::Item.desc(:position).limit(5).scroll(saved_cursor) do |record, _, previous_cursor|
   # each record, one-by-one
-  saved_cursor = next_cursor
+  saved_cursor = previous_cursor
+end
+```
+
+A cursor to get the previous records is available.
```

Comment:
The example below does not show the use of the previous cursor.

Reply:
I tried to clarify the readme.